projects/components/src/utils/test/widget-object.ts

```diff
@@ -49,18 +49,20 @@ export abstract class WidgetObject<T> {
 
     /**
      * Finds first element within this widget matching the given selector
-     * @param cssSelector What to search for
+     * @param selector What to search for
      * @param parent Where to start the search; defaults to the root of this component
      */
-    protected findElement(cssSelector: string, parent: DebugElement = this.root): DebugElement {
-        return parent.query(By.css(cssSelector));
+    protected findElement(selector: string | Type<unknown>, parent: DebugElement = this.root): DebugElement {
```

Comment:
We should have a unit test for this

Reply:
As per our discussion orally, we decided to skip this as we have to create a test directive like  ClickTrackerComponent for testing this function